routers/web/repo/wiki.go

```diff
@@ -598,7 +603,12 @@ func Wiki(ctx *context.Context) {
 func WikiRevision(ctx *context.Context) {
 	ctx.Data["CanWriteWiki"] = ctx.Repo.CanWrite(unit.TypeWiki) && !ctx.Repo.Repository.IsArchived
 
-	if !ctx.Repo.Repository.HasWiki() {
+	hasWiki, err := gitrepo.IsWikiRepositoryExist(ctx, ctx.Repo.Repository)
+	if err != nil {
+		ctx.ServerError("IsWikiRepositoryExist", err)
+		return
+	}
+	if !hasWiki {
```

Comment:
looks in previous design, error from `util.IsDir` was ignored with a log be added, are you sure it's necessary to change this logic to return an 500 error?

Reply:
`util.IsDir` has checked `os.IsNotExist(err)`, so I don't think the error should be ignored.